docusaurus/docs/Android/02-client/06-guides/11-video-agora.mdx

```diff
@@ -0,0 +1,798 @@
+# Agora: Video Integration Guide for Android
+
+This page shows you how to integrate [Agora](https://www.agora.io/en/) into Stream Chat SDK for implementing video call features.
+
+## Introduction
+Video calls have become immensely popular since the onset of the pandemic. Today, we take a look at how you can use the service of [Agora](https://www.agora.io/en/) to integrate video calls into the Stream Chat SDK.
+
+Agora is an infrastructure provider for live, interactive voice and video. They offer native SDKs for mobile platforms, cross-platform frameworks, and the web and allow for simple integration with very few lines of code. They cover a wide range of use cases such as video conferencing, interactive live-streaming, real-time messaging, and many more.
```

Comment:
It would be nice to make the word `Agora` in bold or italic of in code to differentiate it from the other word. Agora means "now" in portuguese and may have confuse some readears.

Reply:
done: [6e70f22](https://github.com/GetStream/stream-chat-android/pull/4205/commits/6e70f220cf1d7c2124649b896c7536087a448eef)